wiki/Tournaments/UKCC/X/en.md

```diff
@@ -6,13 +6,610 @@ tags:
+
+### Qualifiers
+
+- Players will play each map in the Qualifiers mappool once.
+- There will be 3 NoMod, 2 Hidden, 2 Hard Rock and 2 Double Time maps.
+- The top 32 scores will progress into the Swiss bracket.
+
+### Swiss
+
+- Since Swiss bracketing is a natural seeding system, matchups will be drawn between **top 16 and bottom 16 players randomly** to begin.
+- Two matches will happen every weekend for each player, one on Saturday and one on Sunday - therefore, **two matchups will be drawn per weekend** for each player.
+- The order the matches are played in **does not matter**.
+- Matchups will be drawn between players with the **same aggregate match W-L-D score** (eg. players at 3-0-1 will prioritise playing against the players with the most similar W-L-D), and will prioritise **avoiding rematches** until each active player has played each other once.
+
+### Rolling
+
+- When both players are in the lobby and ready, the referee will ask the players to `!roll`. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).
+
+### Banning
+
+- Each player will have **one** ban.
+- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
+- There are no restrictions on bans and picks. Players are free to ban/pick any map regardless of mod.
+
+### Warmups
+
+- There will be **no** warmups played.
+
+### Picking
+
+- Players have **90 seconds** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
+- **Failed scores will count, and every map will be played with the NF modification enabled**
+- Each player picks **four times**. Once 8 maps have been picked and played, the match ends.
+- There are **no Tiebreakers.**
+
+### Health system
+
+- Each player who qualifies into Swiss will **begin with 8,000,000 HP**, which carries through matches.
+- During a match, the **losing player of each map pick** will **lose HP equal to the score difference** between the two players.
+- If a player’s HP reaches 0, they are **immediately eliminated from the tournament**, and if their match is still in progress it ends immediately.
+
+### Pocket Picks
+
+- When signing up to the tournament, each player will **submit three maps** to be listed as “Pocket Picks” for use during Swiss.
+- Pocket Picks can be **any map, with any mods**. The mods chosen are forced.
+- While there is room to push boundaries, the **rule of common sense** applies when submitting Pocket Picks - the maps will be reviewed case by case, and if a map is too far out of the playable range it will be rejected and a new Pocket Pick will have to be submitted.
+- Only up to one Pocket Pick can be played per match, and only as a player’s **3rd or 4th pick during the match**.
+- Once a Pocket Pick has been played, it **cannot be picked again for the rest of the tournament**.
+- Pocket Picks cannot be banned.
+
+### Byes
```

Comment:
even after reading these points, i still have absolutely no idea what a "bye" is

Reply:
Would a rewording of the bullet points to the following be better at helping understand what a "Bye" is?

```
- A player receives a bye when an opponent is eliminated from the tournament before the scheduled match can take place.
- If a player forfeits a match, both players will receive a bye for that match.
- If a player receives a bye, once all scheduled matches for each round are completed, the player will receive an HP deduction of 90% of the average HP lost by the players at their general aggregate match score (+/- 1).
```

Reply:
ah, so this functions as a sort of passive hp drain to account for matches that weren't played

can the last point be reworded as

```
- If a player receives a bye, once all scheduled matches for a round are completed, the player will receive an HP deduction of 90% of the average HP lost by players with similar aggregate match scores (+/- 1).
```

i assume this is calculated after every round since you used "each". are the byes counted individually? so if you receive two byes in the same round, you'd lose this 90% twice?

does "aggregate" mean that the match score counted here is the total match score for the stage, or is it just for the round?

Reply:
Committed (https://github.com/ppy/osu-wiki/pull/7272/commits/2ef651d242bf2ede9a99eb38204bcfe4f19b105c) the changes discussed here with some minor clarifications, if there are any changes that you think are required let me know.